server/block/cactus.go

```diff
@@ -75,6 +61,34 @@ func (c Cactus) RandomTick(pos cube.Pos, w *world.World, r *rand.Rand) {
 	w.SetBlock(pos, Cactus{Age: c.Age}, nil)
 }
 
+// CanGrowHere implements logic to check if cactus can live/grow here
+func (c Cactus) CanGrowHere(pos cube.Pos, w *world.World) bool {
```

Comment:
Out of scope?

Reply:
I wanted to D.R.Y.-up the placement logic from the UseOnBlock, NeighbourUpdateTick and RandomTick functions (for both Cane and Cactus). 
@JustTalDevelops I can remove it if that is preferred.

Reply:
Oh that's fair enough, ignore this.